src/web/controllers/user/screen_user_controller.py

```
@@ -408,3 +423,20 @@ async def htmx_user_rotator(
         if web_context.error:
             return web_context.error
         return self._user_screen_render(web_context)
+
+    @head(
+        path=[
```

The interest of implementing the HEAD method is to suppress exceptions fro the log but it does not solve the initial problem: rotators and screens are not correctly displayed on JS-less clients.